### PR TITLE
Remove overcomplete tests

### PR DESCRIPTION
**Pull request by SpacePossum**

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
I think it's fine to have extra tests even if they basically duplicate another one, so long as there's not too many.

**Comment by SpacePossum** [comment]
they do not test something similar, they test the exact same thing

File: tests/Fixer/ArrayNotation/TrimArraySpacesFixerTest.php [hunk 1]
```diff
@@ -246,14 +246,6 @@ public function provideFixCases()
             ),
 
             array(
-                '<?php $foo = array(array("foo"), array("bar"));',
-            ),
-
-            array(
-                '<?php $foo = [["foo"], ["bar"]];',
```
Comment (keradus) [inline tests/Fixer/ArrayNotation/TrimArraySpacesFixerTest.php]: where it's duplicated ?

**Comment by keradus** [comment]
while it makes sense, i hope that you did not look for it manually...

File: tests/Fixer/ArrayNotation/TrimArraySpacesFixerTest.php (hunk 1 above)
Comment (SpacePossum) [inline tests/Fixer/ArrayNotation/TrimArraySpacesFixerTest.php]: https://github.com/SpacePossum/PHP-CS-Fixer/blob/97cac7f208f934333aa1d31ad8e43cd90d993d6a/tests/Fixer/ArrayNotation/TrimArraySpacesFixerTest.php#L154 
https://github.com/SpacePossum/PHP-CS-Fixer/blob/97cac7f208f934333aa1d31ad8e43cd90d993d6a/tests/Fixer/ArrayNotation/TrimArraySpacesFixerTest.php#L224
https://github.com/SpacePossum/PHP-CS-Fixer/blob/97cac7f208f934333aa1d31ad8e43cd90d993d6a/tests/Fixer/ArrayNotation/TrimArraySpacesFixerTest.php#L234
https://github.com/SpacePossum/PHP-CS-Fixer/blob/97cac7f208f934333aa1d31ad8e43cd90d993d6a/tests/Fixer/ArrayNotation/TrimArraySpacesFixerTest.php#L244
I think

**Comment by SpacePossum** [comment]
>you did not look for it manually... 

I found those while playing around with a test-test-tool, while most of these are harmless there was one nice find I think : )

**Comment by keradus** [comment]
Thank you @SpacePossum.

**Pull request merged**